_posts/2025-04-29-homebrew-4.5.0.md

```diff
@@ -0,0 +1,110 @@
+
+#### Deprecations
+
+- [Homebrew 4.5's various deprecations/disables/removals](https://github.com/Homebrew/brew/pull/19801)
+- [The Ubuntu 18 Docker image has been removed](https://github.com/Homebrew/brew/pull/19190)
+
+Other changes since 4.4.0 I'd like to highlight are the following:
+
+- [`brew install --ask` and `HOMEBREW_ASK` allow viewing the packages, dependencies and sizes in a prompt before installation](https://github.com/Homebrew/brew/pull/19254)
+- [`brew install --skip-link` allows installation without running `brew link`](https://github.com/Homebrew/brew/pull/19136)
+- [`brew update-if-needed` provides a much faster possible replacement for `brew update` that does nothing if no auto-update is required](https://github.com/Homebrew/brew/pull/19626)
+- [`brew install --as-dependency` allows installation of formulae as dependencies rather than "on request"](https://github.com/Homebrew/brew/pull/19570)
+- [`brew` allows being run by `root` in `podman` containers](https://github.com/Homebrew/brew/pull/18574)
+- [`HOMEBREW_TEMP` defaults to `/var/tmp` on Linux, assuming it exists, is readable and is writable](https://github.com/Homebrew/brew/pull/19824)
+- [`brew install --cask` produces fewer GitHub Actions warnings](https://github.com/Homebrew/brew/pull/19009)
+- [Homebrew supports GCC 15](https://github.com/Homebrew/brew/pull/19811)
+- [`brew pyenv-sync` creates major version symlinks to fix `pyenv` support](https://github.com/Homebrew/brew/pull/18978)
+- [The Homebrew macOS `.pkg` installer will upgrade existing installations](https://github.com/Homebrew/brew/pull/19406)
+- [`HOMEBREW_UPGRADE_GREEDY_CASKS` allows specifying a list of casks that should always be upgraded with `--greedy`](https://github.com/Homebrew/brew/pull/19375)
+- Formulae can include
+  [PowerShell (`pwsh`)](https://github.com/Homebrew/brew/pull/19407)
+  or
+  [`clap`](https://github.com/Homebrew/brew/pull/18945)
+  completions.
+- [`@@HOMEBREW_PREFIX@@` can be replaced with `HOMEBREW_PREFIX` in external patches](https://github.com/Homebrew/brew/pull/18613)
+- [`brew alias` and `brew unalias` commands are part of Homebrew/brew rather than an external tap](https://github.com/Homebrew/brew/pull/19258)
+- [`brew edit` and `brew bundle edit` look for VSCode variants e.g. Cursor](https://github.com/Homebrew/brew/pull/19545)
+- [`brew bump*` can bump synced formula together](https://github.com/Homebrew/brew/pull/19798)
+- [`brew *env-sync` has a `HOMEBREW_ENV_SYNC_STRICT` mode for stricter version handling](https://github.com/Homebrew/brew/pull/19577)
+- In formulae and casks,
+  [`deprecate!`/`disable` support specifying replacement software](https://github.com/Homebrew/brew/pull/18733)
+  and
+  [can specify replacement type](https://github.com/Homebrew/brew/pull/19578)
+- [`brew bump-*` only warns, rather than error, on duplicate PRs for non-official taps](https://github.com/Homebrew/brew/pull/19670)
```

Comment:
```suggestion
- [`brew bump-*` only warns, rather than errors, on duplicate PRs for non-official taps](https://github.com/Homebrew/brew/pull/19670)
```